Procfile

```diff
@@ -0,0 +1,2 @@
+worker: go run ./main.go
```

Comment:
@junan Where will you use this one?

Reply:
It will run the worker, https://github.com/junan/google-scraper/blob/develop/worker/main.go

Reply:
Where do you use this `Procfile`?
In the `Makefile` you use `go run worker/main.go` to start worker and `bee run` to start the application.

Reply:
@gutakk

Reply:
My `Dockerfile.worker` ENTRYPOINT is  `["/app/worker/worker"]`. So it runs the the workers `main.go`

Reply:
As discussed in slack, Procfile is used on Heroku only.